server/cf_action.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2020 Google LLC
```

Comment:
Since you are reorganizing, can the cloud function, and cloud function related, files be moved into a subdirectory of server named cloud_function, or something similar of your choice?  And anything that is shared between them moved to a subdirectory called common or something similar?

Reply:
I don't know how to do that and make it work.